docs/tutorials/aws.md

```diff
@@ -48,15 +49,19 @@ spec:
     spec:
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.2.1
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.3.0
```

Comment:
or we can leave this open until v0.3.0 is released and we won't need to update docs twice